doctr/datasets/iiit5k.py

```diff
@@ -73,24 +81,26 @@ def __init__(
             if not os.path.exists(os.path.join(tmp_root, _raw_path)):
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, _raw_path)}")
 
+            if use_polygons:
```

Comment:
That wouldn't work because with `recognition_task=True` we "crop on the fly" this works with `use_polygons=True | False` but we need the coordinates to crop at this point ^^